ec2/sgs.go

```diff
@@ -0,0 +1,72 @@
+package ec2
+
+import (
+	"context"
+	"github.com/YaleSpinup/apierror"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	log "github.com/sirupsen/logrus"
+)
+
+// ListSecurityGroups List all security groups in an aws account
+func (e *Ec2) ListSecurityGroups(ctx context.Context, org string) ([]map[string]*string, error) {
+	log.Infof("listing ec2 security groups (org: '%s')", org)
+
+	var filters []*ec2.Filter
+	if org != "" {
+		filters = []*ec2.Filter{inOrg(org)}
+	}
+
+	input := ec2.DescribeSecurityGroupsInput{
```

Comment:
I don't think you need this `input` variable here, it can just go inline with the call below